components/ide/gha-update-image/lib/jb-helper/jb-gradle-task-config.ts

```diff
@@ -35,7 +38,7 @@ export const latestGatewayPluginGradleTarget: TaskInfo = {
     productId: "gateway",
     productCode: "GW",
     productType: "eap,rc,release",
-    usePlatformVersionType: "version",
+    usePlatformVersionType: "build",
```

Comment:
I see we are already using this:
https://github.com/gitpod-io/gitpod/blob/babc57127451d365ac017a78a6094ae12eeab7f2/components/ide/jetbrains/gateway-plugin/gradle-latest.properties#L11

So it makes sense to be on build. Thanks for the explanation below!